website/content/api-docs/query.mdx

```diff
@@ -11,7 +11,7 @@ The `/query` endpoints create, update, destroy, and execute prepared queries.
 Prepared queries allow you to register a complex service query and then execute
 it later via its ID or name to get a set of healthy nodes that provide a given
 service. This is particularly useful in combination with Consul's
-[DNS Interface](/docs/discovery/dns) as it allows for much richer queries than
+[DNS Interface](/docs/discovery/dns#prepared-query-lookups) as it allows for much richer queries than
```

Comment:
```suggestion
[DNS Interface](/docs/discovery/dns#prepared-query-lookups) because it enables more specific queries than
```
We might as well update the description with more concrete language while we're at it.